build-logic/convention-plugins/src/main/kotlin/social.androiddev.library.common.gradle.kts

```diff
@@ -0,0 +1,62 @@
+plugins {
```

Comment:
I like the idea of having a top level comment on these different plugins with a description of what they are for and/or do. Mind adding that to the three you created?